website/docs/03_components/02_package-repository.md

```diff
@@ -1,3 +1,7 @@
 # Package Repository
```

Comment:
filename should be 03_package-repository

Reply:
fixed in https://github.com/glasskube/glasskube/pull/161